build_image.sh

```diff
@@ -111,7 +111,7 @@ if [ "$IMAGE_TYPE" = "onie" ]; then
     mkdir -p `dirname $OUTPUT_ONIE_IMAGE`
     sudo rm -f $OUTPUT_ONIE_IMAGE
 
-    generate_device_list "./installer/$TARGET_PLATFORM/platforms_asic"
```

Comment:
sonic-utilities also need to change if the path change.
https://github.com/Azure/sonic-utilities/blob/master/sonic_installer/bootloader/grub.py#L21